opensearch/opensearch-kafka/readme.md

```diff
@@ -0,0 +1,55 @@
+# OpenSearch Example
+
+This repo is intended to provide the SA team with a means to populate an OpenSearch index with data so that you can demonstrate how OpenSearch can be used.
+
+## Requirements
+
+- [Terraform CLI](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli)
+- Python 3.8+
+- [Aiven CLI](https://docs.aiven.io/docs/tools/cli.html)
+- Project created via Aiven Console or CLI for running services
+- (Optional) [Postman v2.1](https://www.postman.com/downloads/)
+
+
+## Steps
+1. Run the setup.sh file to unzip the file ``` ./setup.sh ```
+2. Open a new terminal/shell window and run the following command to save the Aiven API token to an ENV variable
+```
+. ../scripts/save-token.sh
+```
```

Comment:
I recently discovered an even more automated way to skip having to save the token, you can see the example [here](https://github.com/aiven/aiven-examples/blob/main/kafka/kafka-streams/terraform/data.tf)

Reply:
awesome, have updated example to use this